extensions/opentelemetry/deployment/src/test/java/io/quarkus/opentelemetry/deployment/common/traces/NoResource.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.opentelemetry.deployment.common.traces;
+
+import io.quarkus.opentelemetry.runtime.tracing.Traceless;
+
+@Traceless
+public class NoResource {
```

Comment:
@brunobat This one do not throws an exception, because we have the following rule:

```java
                    if (Objects.isNull(possibleClassAnnotatedWithPath)) {
                        continue;
                    }
```

Reply:
yeah, better throw a precise exception because their code will not behave as expected.